@in static check CI, mypy is throwing the same error which pylint is set to ignore. Can we ignore this mypy error?

```
airflow/providers/snowflake/hooks/snowflake.py:55: error: "SnowflakeHook" has
no attribute "snowflake_conn_id"
            conn = self.get_connection(self.snowflake_conn_id)  # pylint: ...
                                       ^
airflow/providers/snowflake/hooks/snowflake.py:129: error: "SnowflakeHook" has
no attribute "snowflake_conn_id"
            if self.snowflake_conn_id:  # pylint: disable=no-member
               ^
airflow/providers/snowflake/hooks/snowflake.py:130: error: "SnowflakeHook" has
no attribute "snowflake_conn_id"
    ...         connection_object = self.get_connection(self.snowflake_conn_i...
```
Yes. These errors can be ignored by `typing: ignore` because we're sure these attributes exist but are dynamically created.

--- a/airflow/providers/snowflake/hooks/snowflake.py
+++ b/airflow/providers/snowflake/hooks/snowflake.py
@@ -52,7 +52,7 @@ class SnowflakeHook(DbApiHook):
         One method to fetch connection params as a dict
         used in get_uri() and get_connection()
         """
-        conn = self.get_connection(self.snowflake_conn_id)  # pylint: disable=no-member
+        conn = self.get_connection(self.snowflake_conn_id)  # type: ignore # pylint: disable=no-member
         account = conn.extra_dejson.get('account', '')
         warehouse = conn.extra_dejson.get('warehouse', '')
         database = conn.extra_dejson.get('database', '')
@@ -126,8 +126,10 @@ class SnowflakeHook(DbApiHook):
 
         intended to be used by external import and export statements
         """
-        if self.snowflake_conn_id:  # pylint: disable=no-member
-            connection_object = self.get_connection(self.snowflake_conn_id)  # pylint: disable=no-member
+        if self.snowflake_conn_id:  # type: ignore  # pylint: disable=no-member
+            connection_object = self.get_connection(
+                self.snowflake_conn_id  # type: ignore  # pylint: disable=no-member
+            )
             if 'aws_secret_access_key' in connection_object.extra_dejson:
                 aws_access_key_id = connection_object.extra_dejson.get('aws_access_key_id')
                 aws_secret_access_key = connection_object.extra_dejson.get('aws_secret_access_key')

--- a/airflow/providers/snowflake/hooks/snowflake.py
+++ b/airflow/providers/snowflake/hooks/snowflake.py
@@ -52,7 +52,7 @@ class SnowflakeHook(DbApiHook):
         One method to fetch connection params as a dict
         used in get_uri() and get_connection()
         """
-        conn = self.get_connection(self.snowflake_conn_id)  # type: ignore # pylint: disable=no-member
+        conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined] # pylint: disable=no-member
         account = conn.extra_dejson.get('account', '')
         warehouse = conn.extra_dejson.get('warehouse', '')
         database = conn.extra_dejson.get('database', '')
@@ -126,9 +126,9 @@ class SnowflakeHook(DbApiHook):
 
         intended to be used by external import and export statements
         """
-        if self.snowflake_conn_id:  # type: ignore  # pylint: disable=no-member
+        if self.snowflake_conn_id:  # type: ignore[attr-defined]  # pylint: disable=no-member
             connection_object = self.get_connection(
-                self.snowflake_conn_id  # type: ignore  # pylint: disable=no-member
+                self.snowflake_conn_id  # type: ignore[attr-defined]  # pylint: disable=no-member
             )
             if 'aws_secret_access_key' in connection_object.extra_dejson:
                 aws_access_key_id = connection_object.extra_dejson.get('aws_access_key_id')

--- a/airflow/providers/snowflake/hooks/snowflake.py
+++ b/airflow/providers/snowflake/hooks/snowflake.py
@@ -52,7 +52,9 @@ class SnowflakeHook(DbApiHook):
         One method to fetch connection params as a dict
         used in get_uri() and get_connection()
         """
-        conn = self.get_connection(self.snowflake_conn_id)  # type: ignore[attr-defined] # pylint: disable=no-member
+        conn = self.get_connection(
+            self.snowflake_conn_id  # type: ignore[attr-defined] # pylint: disable=no-member
+        )
         account = conn.extra_dejson.get('account', '')
         warehouse = conn.extra_dejson.get('warehouse', '')
         database = conn.extra_dejson.get('database', '')

